pyscriptjs/src/main.ts

```diff
@@ -309,7 +313,7 @@ modules must contain a "plugin" attribute. For more information check the plugin
 
     // lifecycle (7)
     executeScripts(runtime: Runtime) {
-        this.PyScript = make_PyScript(runtime);
+        this.PyScript = make_PyScript(runtime, this);
```

Comment:
Do you think we should add a comment mentioning that `this = PyScriptApp`? Just wonder if that makes it easier to know what we are passing (although one could check the parameter name as well)

Reply:
Agreed.